src/features/LoginModal/CreateAccountForm/CreateAccountForm.tsx

```diff
@@ -39,6 +39,7 @@ const CreateAccountForm: React.FC<CreateAccountFormProps> = (props) => {
             updateProfile(createdUser.user, {displayName: displayName})
             loginUser.mutate({id: createdUser.user.uid, email: email, displayName: displayName}, {
                 onError: (err: any) => {
+                    setError(err.response.errors[0].message || 'Something went wrong, please try again or contact us for help.')
```

Comment:
At the moment, the backend doesn't throw an error when signing in under an account that doesn't exist, instead it just creates it. Should have this as a signUp mutation instead of login mutation